### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by 0xwuchuan**

### CODE REVIEW

**Comment by 0xwuchuan** [comment]
TLDR
- Default Network now is Rinkeby, to change to local network(Ganache) look at README for instructions
- quote system works now with Quote event -> Have to change back to quote implementation and insert argument in mintTokens()
- new function getQuote() in contract.js returns quote when supplied with tokenId -> not sure how to implement it on profile and gallery
- new function checkNetwork() implemented in interact.js that checks for user in Rinkeby network if not mint button and tokensRemaining wont work -> in Mint.js useEffect() for now, not sure whether there is a better place for it
- Updated README for instructions on how to setup local env or deploy to rinkeby
